integration-cli/docker_cli_secret_create_test.go

```diff
@@ -104,3 +108,33 @@ func (s *DockerSwarmSuite) TestSecretCreateResolve(c *check.C) {
 	c.Assert(out, checker.Not(checker.Contains), id)
 	c.Assert(out, checker.Not(checker.Contains), fake)
 }
+
+func (s *DockerSwarmSuite) TestSecretCreateWithFile(c *check.C) {
```

Comment:
In the end there could be a unit test for that. But I guess there was no *cli integration test* on `docker secret create` so, I'm ok with that :angel: